### DISCUSSION TITLE
Previously installed packages missing in 2nd phase

### DISCUSSION THREAD

**Discussion by shaido987** [PaddlePaddle/PaddleSpatial#218] [BaiduKDDCup2022] [answered]
Hello,

When submitting to the 2nd phase test data I get the following error:

> Err: IMPORT ERROR: No module named 'darts', load module [path: /tmp/tmpx6f0jdc2/...

The darts package was installed and working correctly in the phase 1 submissions in the same environment.

Environment: pytorch
Team: didadida_hualahuala
Time: 2022-06-23 14:28

**Comment by Ethan1X** [comment]
We also encountered similar problems.
When submitting to the 2nd phase test data I get the following error:
`Err: IMPORT ERROR: No module named 'pgl', load module [path: /tmp/tmp65que2tz/test-lxf/predict.py]!!`
The pgl was installed and working correctly in the phase 1 submissions in the same environment.
Environment: paddlepaddle
thank u.

**Reply by Ethan1X** [reply]
The second time I resubmitted it, it returned to normal.

**Comment by shaido987** [comment, marked as answer]
This seems to have been fixed. The submission I made was automatically re-evaluated and returned a normal result without error.
